src/main.ts

```diff
@@ -168,6 +168,33 @@ export class Main {
       this.triggerUpdateCheck();
       this.setupUpdateInterval();
     }
+
+    // Make sure we have a ~/.chef directory
+    this.createChefDir();
+  }
+
+  // make the ~/.chef directory if it doesn't exit and add a sample credentials file
+  private createChefDir() {
+    let os = require('os'),
+    fs = require('fs'),
+    path = require('path');
+    let chefDir = os.homedir() + path.sep + '.chef';
```

Comment:
`path.join(os.homedir(), '.chef')` 

Which led me see we're already doing some directory creation over in https://github.com/chef/chef-workstation-app/blob/master/src/app-config/app-config.ts.  I wonder if it wouldn't be better to house this function there?

Reply:
That is much simpler. Thanks